### ISSUE TITLE
Property:degreeOfEstablishment

### ISSUE THREAD

**Issue by gdadade** [tdwg/cd#276]
see https://dwc.tdwg.org/list/#dwc_degreeOfEstablishment

I think this is important both on specimen and collection level, e.g. collection holds (dead) preserved specimens from Zoos, Botanic Gardens, Culture Collections, agriculture etc.

BasisOfRecord:LivingSpecimen does not work for dead organisms (or tissue/DNA samples) originating from ex situ environments

Could be used in combination with dwc:pathway, but this might differ within one collection

**Comment by debpaul** [comment]
> Could be used in combination with dwc:pathway, but this might differ within one collection

I do think @gdadade that providing a single value for a Collection Description for this field might not be possible. I'm assuming the field expects a single value? @qgroom @gdadade ?

**Comment by qgroom** [comment]
Indeed, our living collection actually has an independent reference collection created from cultivated plants. I'm not entirely convinced it is that useful or that the vocabulary from dwc:degreeOfEstablishment is suitable, but it deserves some consideration.

**Comment by gdadade** [comment]
@debpaul you are right, on a collection level we might need something like "containsExSituSpecimens"
